tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/NewLiferayWorkspaceOp.java

```diff
@@ -84,6 +87,66 @@
     @Service( impl = NewLiferayWorkspaceServerNameService.class )
     ValueProperty PROP_SERVER_NAME = new ValueProperty( TYPE, BaseLiferayWorkspaceOp.PROP_SERVER_NAME );
 
+    // *** Environment ***
+
+    @Label( standard = "environment" )
+    @DefaultValue( text = "local" )
+    ValueProperty PROP_ENVIRONMENT = new ValueProperty( TYPE, "Environment" );
+
+    Value<String> getEnvironment();
+
+    void setEnvironment( String environment );
+
+    // *** ModulesDefaultRepositoryEnabled ***
+
+    @Type( base = Boolean.class )
+    @DefaultValue( text = "false" )
+    @Label( standard = "modules default repository enabled" )
+    ValueProperty PROP_MODULES_DEFAULT_REPOSITORY_ENABLED =
```

Comment:
This is confusing in the UI..  i don't exactly know what this property is doing.  Also it should be lower down in the UI, maybe at the very end.